README.md

```diff
@@ -0,0 +1,499 @@
+
+> We could try to run the postcss and tailwind plugin directly in the browser, making it to work with node polyfills, using tailwind internals implementations like they do in [play.tailwindcss.com](https://play.tailwindcss.com/), or using the brand new [Web Containers](https://blog.stackblitz.com/posts/introducing-webcontainers/) but for simplicity's sake, we do it in a simple node service.
+>
+
+## Creating the project
+
+Let’s create the new project called `tailwind-editor` with [Vite](https://vitejs.dev/) running `npm create`. I’m going to use [Vue](https://vuejs.org/) for the Frontend because I’m more comfortable with it and also because it is Awesome ;)
+
+```bash
+$ dev npm create vite@latest
+✔ Project name: … tailwind-editor
+✔ Select a framework: › Vue
+✔ Select a variant: › JavaScript
+```
+
+Then, we add the dependencies for the service.
+
+```bash
+$ cd tailwind-editor
+$ npm install --save express cors postcss tailwindcss
+```
+
+The resulting package:
+
+```json
+{
+  "name": "tailwind-editor",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "cors": "^2.8.5",
+    "express": "^4.18.1",
+    "postcss": "^8.4.17",
+    "tailwindcss": "^3.1.8",
+    "vue": "^3.2.37"
+  },
+  "devDependencies": {
+    "@vitejs/plugin-vue": "^3.1.0",
+    "vite": "^3.1.0"
+  }
+}
+```
+
+## Creating the Tailwindcss service
+
+Now, we are going to create the service that will receive the Tailwindcss config and return the CSS. Let’s start with a file `src/tailwind-as-a-service.js` which will contain the [ExpressJs](https://expressjs.com/) server with the `cors` middleware to support [cross-origin](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) calls. It is listening by port 8080 to any request to the root path with a `GET` method and returns a text with `Hello World`.
+
+```jsx
+// src/tailwind-as-a-service.js
+
+import express from 'express';
+import cors from 'cors';
+
+const app = express()
+app.use(cors());
+const port = 8080
+
+app.get('/', (req, res) => {
+  res.send('Hello World!')
+})
+
+app.listen(port, () => {
+  console.log(`Tailwind as a service listening on port ${port}`)
+})
+```
+
+Running the server with node we have the response directly in the browser:
+
+```bash
+$ node ./src/tailwind-as-a-service.js
+```
+
+![Hello World](images/hello-world.png)
+
+> We are using Node 16, which supports [ES modules](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules) by default. If you are using older versions of Node you can use CommonJs modules and just rename the file with the `.cjs` extension.
+>
```

Comment:
I think the last sentence could be misleading of confusing. Someone could understand that renaming the file into `.cjs` is enough. Besides, Vite requires node `14.18+` and that version already supports es modules. Maybe would be better to specify a min version of node to follow this article and the demo. Something like:

Since we are using Vite and ES modules, the minimum node version required to follow this article is `14.18+`